usaspending_api/accounts/views/federal_accounts_v2.py

```diff
@@ -674,7 +688,9 @@ def post(self, request, format=None):
                 account_number=F("federal_account_code"),
                 budgetary_resources=Subquery(
                     AppropriationAccountBalances.objects.filter(
-                        final_of_fy=True,
+                        submission__reporting_fiscal_year=fy,
```

Comment:
I wonder if we need both this and line 694:
```
submission__reporting_period_start__fy=fy,
```
Could this be limited to one or the other?

Reply:
I meant to remove one of these. We use `reporting_fiscal_year` on line 508 to get the submissions for the treasury accounts and `reporting_period_start__fy` was already a filter on the AppropriationAccountBalance object before my change.

So is there any strong case for using one over the other? The only real difference I can see is that `reporting_fiscal_year` is always going to be an integer that represents the fiscal year whereas `reporting_period_start__fy` is a date column that we call the `FY()` database function on to get the appropriate fiscal year value.

It seems like using `reporting_fiscal_year` would prevent an unnecessary DB function call, but either seems like it would work.

Reply:
I removed the check on line 694 and kept `submission_reporting_fiscal_year`. If there's a reason to use `reporting_period_start__fy` instead then just let me know and I can change that.

Reply:
This solution sounds good. Thank you!